dataland-inbound-proxy/templates/http-dataland.template

```diff
@@ -19,7 +19,7 @@ server {
 
     # locations
     include                 utils/locations/frontend${FRONTEND_LOCATION_CONFIG}.conf;
-    include                 utils/locations/backend${BACKEND_LOCATION_CONFIG}.conf;
+    include                 utils/locations/backend${BACKEND_LOCATION_CONFIG}[.]conf;
```

Comment:
Was macht diese Änderung?

Reply:
Soweit ich das verstanden habe sorgt es dafür, dass nginx keinen Fehler wirft wenn es das File an der gegebenen Stelle nicht findet. Das wäre in unserem Fall immer dann wenn BACKEND_LOCATION_CONFIG nicht den Wert Dev hat.